### PR TITLE
Correct MQTT fan breaking changes info

### PR DESCRIPTION
**Pull request by emontnemery**
## Proposed change

Correct MQTT fan breaking changes info

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by emontnemery** [comment]
@jbouwh Please review

File: source/_posts/2021-04-07-release-20214.markdown [hunk 1]
```diff
@@ -951,7 +951,7 @@ Home Assistant Core 2021.7.0:
 
 - `speed_command_topic`
 - `speed_state_topic`
-- `state_value_template`
+- `speed_value_template`
 - `speed_list`
```
Comment (jbouwh) [inline source/_posts/2021-04-07-release-20214.markdown]: The MQTT fan configuration option for `speed_list` attribute is `speeds`.
Further the following configuration options won't be needed anymore:
- `payload_off_speed`
- `payload_low_speed`
- `payload_medium_speed`
- `payload_high_speed`

The rest LGTM

**Review by jbouwh — CHANGES_REQUESTED** [review]
Please list the  `speeds` config option and add the speed payload specific config options to the list of options to be removed.

**Review by jbouwh — APPROVED** [review]
LGTM

**Review by jbouwh — APPROVED** [review]

**Review by frenck — APPROVED** [review]

**Pull request merged**